libarchive/archive_read_private.h

```diff
@@ -181,6 +181,10 @@ struct archive_read {
 	/* File offset of beginning of most recently-read header. */
 	int64_t		  header_position;
 
+	/* Offset of the farthest read header. Used for counting files while
+	 * seeking between headers */
+	int64_t		  max_header_position;
```

Comment:
What is this for?  It's not actually used anywhere.

Reply:
This is a bug.
I should move `++_a->file_count;` into the if block.